ExerciseTwo/exercise.html

```diff
@@ -1,9 +1,4 @@
-<!-- 
-    INSTRUCTIONS:
-    Fix the 3 lines with indentation issues below.
--->
-    <!DOCTYPE html>
-  <head>
-    
-      </head>
-    </html>
+<!DOCTYPE html>
+<html>
```

Comment:
`<!DOCTYPE html>` and `<html>` serve the same purpose in an html document.
They both indicate that this is an html doc.
You can take out `<html>`.
`<!DOCTYPE html>` is better.